src/AppRouter.tsx

```diff
@@ -11,9 +11,14 @@ export const AppRouter = () => {
   return (
     <Switch>
       <Route path="/" exact>
-        {/* TODO: Break into a view component*/}
-        <Blockly />
-        <RobotSimulator />
+        <div className="grid-container">
+          <div className="menu-bar"></div>
```

Comment:
Since the menu bar is usually static on website (always present on the top), I think we should move this to the `App` component so that it lives alongside AppRouter. So it would look like:
```
App
  Header
  Router
    Simulator & subcomponents
  Footer
```
This will prevent the header and footer from re-rendering / needing to be imported into every page in the future.